otter/convergence/planning.py

```diff
@@ -186,29 +186,22 @@ def _converge_lb_state(server, current_lb_nodes):
         if desired.equivalent_definition(node.description)]
 
     if desired_matching_existing:
-        met_desireds, good_nodes = zip(*desired_matching_existing)
```

Comment:
Non-blocking, and not even really about this change: Personally I dislike the use of `_` as a variable name. I think it's an unnecessary and confusing idiom.  This is not really a comment on this change, but using `zip` in this way when you expect a fixed number of results is weird.

Rather than make up a name for this variable though, you could just compress this whole conditional suite into:

```python
met_desireds = next(iter(zip(*desired_matching_existing), ()))
```